SdmxStructureParser/src/main/java/org/sdmxsource/sdmx/structureparser/builder/sdmxbeans/impl/DataStructureDTO.java

```diff
@@ -284,6 +284,11 @@ public static class Attributerelationship {
          * The Primary measure.
          */
         public String primaryMeasure;
+
+        /**
+         * None relationship
```

Comment:
imo, it makes sense to update comment to directly indicate that this is a dataset relationship